core/quill.js

```diff
@@ -81,6 +82,11 @@ class Quill {
       this.disable();
     }
     this.emitter.emit(Emitter.events.READY);
+    this.emitter.on(Emitter.events.TEXT_CHANGE, (delta) => {
+      if (delta.length() > 0) {
+        this.toggleEmpty(this.getLength() <= 1);
```

Comment:
It would, however it all depends on your IE support. http://caniuse.com/#feat=classlist No version of EI uses the boolean toggle arg as of yet, so there still needs a little bit of logic somewhere to decide whether to add/remove the class.